src/renderer/components/Aside/Index.vue

```diff
@@ -59,6 +59,18 @@
           console.log(err)
         })
       }
+    },
+    data: function () {
```

Comment:
I think the computed attribute can be used to replace data here.
```javascript
vibrancy () {
  return is.macOS()
    ? {
      backgroundColor: 'transparent'
    }
    : {}
}
```

Reply:
OK, I'm not familiar with vue. XD